src/sgx/CMakeLists.txt

```diff
@@ -53,8 +53,15 @@ set (WAMR_BUILD_TARGET X86_64)
 # Set WAMR include flags
 # (fast) interpreter, AOT, libc, wasi, and pthread are enabled by default
 
+# Enabling the spec tests performs stricter checks. E.g. it will error if can't
+# link an imported function
+if (FAASM_SGX_DEBUG)
+    add_definitions (-DWASM_ENABLE_SPEC_TEST)
```

Comment:
Adding this definition makes WAMR error out if it fails to link an imported function.

Reply:
I think we should have this on all the time (both SGX and non-SGX WAMR). The WAVM implementation has this strict behaviour.